cwr/api_test.go

```diff
@@ -17,7 +17,7 @@
 //
 // If you have any questions please contact yo@jaak.io
 
-package cwr
+package cwr_test
 
```

Comment:
We moved the setup of the test CWR index into a separate package which is then imported into this test file, so having the test within the `cwr` package would cause a cyclic dependency. Using `cwr_test` also ensures we are testing public functionality rather than implementation details, which I prefer :)